--- drumstick-2/CMakeLists.txt	2020-12-29 15:28:45.554855468 -0500
+++ drumstick-2/CMakeLists.txt-new	2020-12-31 01:04:28.324285651 -0500
@@ -124,16 +124,16 @@
 endif()
 
 if(UNIX AND NOT APPLE)
-    find_package(Doxygen REQUIRED dot)
-    if(DOXYGEN_FOUND)
-        string(REPLACE ";" " " DOXYGEN_INCLUDE_DIRS "${Qt5Core_INCLUDE_DIRS}")
-        configure_file("${CMAKE_SOURCE_DIR}/Doxyfile.in"
-            "${CMAKE_CURRENT_BINARY_DIR}/Doxyfile" IMMEDIATE @ONLY)
-        add_custom_target(doxygen
-            ${DOXYGEN_EXECUTABLE} "${CMAKE_CURRENT_BINARY_DIR}/Doxyfile"
-            WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
-    endif()
-    add_subdirectory(doc)
+#    find_package(Doxygen REQUIRED dot)
+#    if(DOXYGEN_FOUND)
+#        string(REPLACE ";" " " DOXYGEN_INCLUDE_DIRS "${Qt5Core_INCLUDE_DIRS}")
+#        configure_file("${CMAKE_SOURCE_DIR}/Doxyfile.in"
+#            "${CMAKE_CURRENT_BINARY_DIR}/Doxyfile" IMMEDIATE @ONLY)
+#        add_custom_target(doxygen
+#            ${DOXYGEN_EXECUTABLE} "${CMAKE_CURRENT_BINARY_DIR}/Doxyfile"
+#            WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+#    endif()
+#    add_subdirectory(doc)
     configure_file(drumstick-alsa.pc.in drumstick-alsa.pc IMMEDIATE @ONLY)
     configure_file(drumstick.spec.in drumstick.spec IMMEDIATE @ONLY)
     install(FILES
